lib/node_modules/@stdlib/stats/base/dists/beta/stdev/README.md

```diff
@@ -144,13 +144,101 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.examples -->
 
-<!-- Section to include cited references. If references are included, add a horizontal rule *before* the section. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
```

Comment:
Can we please keep the empty references section at the bottom? No reason to remove it.

Reply:
done